app/main/forms.py

```diff
@@ -2050,7 +2050,7 @@ class AdminEditLetterBrandingSVGUploadForm(StripWhitespaceForm):
     file = VirusScannedFileField(
         "Upload an SVG logo",
         validators=[
-            FileAllowed(["svg"], "SVG Images only!"),
+            FileAllowed(["svg"], "Logo must be an SVG file"),
```

Comment:
```suggestion
            FileAllowed(["svg"], "the logo must be an SVG file"),
```

This doesn't seem to be in the validation audit spreadsheet but this change would make it consistent with the email one.